binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -128,6 +132,14 @@ public void run() {
                                 }
                             }
                         }
+                        
+                        final Config config = fullConfig.getConfig(); 
+                        if (config != null) {
+                            storeConfigValue(getThing().getConfiguration(), DefaultPropertyKey.SERIAL_NUMBER,
+                                    config.getMACAddress().replaceAll(":", ""));
```

Comment:
Btw, on the sticker on the bridge, the serial is also given with colons. So anywhere where it is presented to the user, it has colons - should hence be the same here.